.github/workflows/build-firmware-and-emulator.yml

```diff
@@ -100,7 +100,6 @@ jobs:
         name: ${{ env.EMULATOR_ARTIFACT }}-${{ matrix.os }}-exe
         path: |
           ${{ matrix.emu_binary }}
-          spiffs_image
```

Comment:
I think this should have just been changed to `assets_image`?

Reply:
Oh wait, does this mean there's no more assets directory to ship? Cool!

Reply:
Yep, it's all in the executable now! Downside is you cant quickly do asset swaps without recompilation.